File: 백준/Bronze/2587. 대표값2/대표값2.java
import java.util.*;
import java.io.*;
public class Main {

    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int[] arr = new int[5];
        for(int i=0; i<5; i++){
            arr[i] = Integer.parseInt(br.readLine());
        }
        Arrays.sort(arr);
        int sum = Arrays.stream(arr).sum();
        System.out.println(sum/5);
        System.out.println(arr[2]);
    }
}

File: 백준/Bronze/2587. 대표값2/대표값2.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let inputLine = 1;
let arr = [];
rl.on("line", (line) => {
  if (inputLine >= 1 && inputLine <= 5) {
    arr.push(parseInt(line));
  }
  if (inputLine == 5) {
    arr.sort((a, b) => a - b);
    let avg = arr.reduce((acc, cur) => acc + cur, 0) / 5;
    let mid = arr[2];
    console.log(avg);
    console.log(mid);
    process.exit();
  }
  inputLine++;
});
